### ISSUE TITLE
Set color of Category

### ISSUE THREAD

**Issue by MaxBotta** [xvicmanx/react-blockly-drawer#10]
Hey, I am using your component for a project in university and it works pretty good, thanks! I was wondering though, how you can define a color for a category which gets generated when you define a block. The category will be displayed without any color.

```
const rotate = {
    name: 'rotate',
    category: 'Robot',
    block: {
        init: function () {
            this.jsonInit({
                "type": "rotate",
                "message0": "Rotate %1",
                "args0": [
                    {
                        "type": "field_angle",
                        "name": "DEGREES",
                        "angle": 90
                    }
                ],
                "previousStatement": null,
                "nextStatement": null,
                "colour": 180,
                "tooltip": "",
                "helpUrl": ""
            });
        },
    },
    generator: (block) => {
        const degrees = `'${block.getFieldValue('DEGREES')}'` || '\'\'';
        const code = 'Rotate(' + degrees + ');\n';
        return code;
    },
};
```

If I'm defining the categories like this, it will display a color, but the category is doubled.
```

<Category name="Robot" colour="270">
   <Block type="rotate"></Block>
</Category>
```

<img width="117" alt="bildschirmfoto 2018-05-22 um 17 41 35" src="https://user-images.githubusercontent.com/18328038/40373586-8407e028-5de7-11e8-8b34-30603cd0e99f.png">

**Comment by xvicmanx** [comment]
Thank you very much for the feedback Max. I guess it is not possible for now but I will work on a solution for it. Hopefully, I will be able to have it by the end of the day or tomorrow.

**Comment by xvicmanx** [comment]
Hello Max, I just published a new version `1.6.1` of the package.
Basically, we have a new property that can be passed to the `BlocklyDrawer`.
This property is called `appearance`. It is an object with the field `categories` which is an object whose keys are the names of your custom categories. Each key should contain an object with styling properties such as `colour`, for example.

```
appearance={
        {
          categories: {
            Demo: {
              colour: '270'
            },
          },
        }
      }
```
Let me know if that works for you.

**Comment by MaxBotta** [comment]
That was fast and it works very well. Thank you!
